presto-main/src/main/java/com/facebook/presto/operator/aggregation/DoubleSumAggregation.java

```diff
@@ -25,13 +25,29 @@
     private DoubleSumAggregation() {}
 
     @InputFunction
-    @IntermediateInputFunction
     public static void sum(NullableDoubleState state, @SqlType(StandardTypes.DOUBLE) double value)
     {
         state.setNull(false);
         state.setDouble(state.getDouble() + value);
     }
 
+    @CombineFunction
+    public static void combine(NullableDoubleState state, NullableDoubleState otherState)
+    {
+        if (state.isNull()) {
+            if (otherState.isNull()) {
+                return;
+            }
+            state.setNull(false);
```

Comment:
I wanted to avoid getting uninitialized value because IIRC generated c-tor for State classes does not initialize fields.
Maybe @cberner may comment on this?